lib/components/narrative/default/default-itinerary.js

```diff
@@ -209,8 +233,7 @@ class DefaultItinerary extends NarrativeItinerary {
               const bSelected = this._isSortingOnAttribute(b)
               if (aSelected) return -1
               if (bSelected) return 1
-
-              return a.order - b.order
+              else return a.order - b.order
```

Comment:
Revert this change.

Reply:
I ran prettier at some point which caused these changes. Removed in 513973b